client/src/css/Jessica.css

```diff
@@ -0,0 +1,55 @@
+h2 {
+  font-size: 60;
+  text-align: center;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+.name {
+  text-align: center;
+  color: #fdf1e8;
+  font-size: 70px;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  position: absolute;
+  top: 150px;
+  left: 0;
+  width: 100%;
+}
+.hobbies {
+  padding-left: 5%;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  line-height: 1.8;
+}
+.border {
+  flex: 1;
+  border-style: solid;
+  border-width: 6px;
+  border-color: #287ec7;
+  margin-right: 50px;
+  margin-left: 50px;
+  margin-bottom: 150px;
+  margin-top: 0%;
+  padding-bottom: 2.5%;
+  height: 350px;
+  float: left;
+}
+
+.container {
+  width: 100%;
+  height: auto;
+  padding: 1%;
```

Comment:
because your width is already 100%, the 1% padding (which adds 1% padding on each side) will actually add horizontal scrolling to your page because the total width will actually be 102% of the screen width. one way to fix this is to just drop the width down to 98%